client/src/components/UI/filter/FilterSelect.tsx

```diff
@@ -55,9 +55,13 @@ const FilterSelect: React.FC<multiOptionProps> = ({
     onSelectChange(selectedValues);
   };
 
+  const customSelectStyle = {
+    width: '100%',
+    minWidth:"140px"
+  };
   return (
     <Select
-      style={{ width: 200 }}
+      style={customSelectStyle}
```

Comment:
could you simply add the `minWidth` to `style={{ width:"100%", minWidth: "140px" }}`?

Reply:
done